docs/resources/convert_v1_to_v2_resource_groups.md

```diff
@@ -0,0 +1,251 @@
+---
+subcategory: "Resource Groups"
+layout: "lacework"
+page_title: "Lacework: convert_to_newer_resource_group"
```

Comment:
Why is the title filled with underscores? Also note, if you are going to display this on the sidebar, typically these are commands in terraform you can run. Is there a convert_to_newer_resource_group command that someone created recently?

Reply:
Is it ok if this document becomes a section of the resource_group command document instead of a separate doc? The Terraform docs don't really have space for reference documentation like this for which there is no command.

Reply:
Yeah that's what I was thinking. Or better yet just host it on fortinet docs as and then we can link it in the resource_group command

Reply:
That is better, let me find a spot for it.

Who should review it?